tools/c7n_tencentcloud/c7n_tencentcloud/resources/tag.py

```diff
@@ -24,7 +24,7 @@ class resource_type(ResourceTypeInfo):
             "method": PageMethod.PaginationToken,
             "pagination_token_path": "Response.PaginationToken",
             "limit": {
-                "Key": "MaxResults",
+                "key": "MaxResults",
```

Comment:
its not clear that tag should be a resource type, its doesn't have standalone use as its primarily an augmentation for other resource types, given that this module should be move to parent package and the resource registration removed.

Reply:
aggred, we will make a TODO for removing the registration.